extensions/hawkbit-extension-artifact-repository-mongo/src/main/java/org/eclipse/hawkbit/artifact/repository/MongoDBArtifactStore.java

```diff
@@ -216,15 +216,15 @@ private static String computeSHA1Hash(final InputStream stream, final OutputStre
     }
 
     /**
-     * Maps a list of {@link GridFSDBFile} to paged list of {@link DbArtifact}s.
+     * Maps a list of {@link GridFSDBFile} to paged list of {@link AbstractDbArtifact}s.
      *
      * @param tenant
      *            the tenant
      * @param dbFiles
      *            the list of mongoDB gridFs files.
      * @return a paged list of artifacts mapped from the given dbFiles
```

Comment:
I think the `paged` is on each javadoc, maybe you can double check the javadoc, something historical maybe? :)